work11/index.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+const wrap = document.getElementById("js-wrap")
+const ul = document.getElementById("js-list");
+// const url = "https://myjson.dit.upm.es/api/bins/ほげほげajy3";
+// const url = "https://myjson.dit.upm.es/api/bins/bu5z";
+const url = "https://myjson.dit.upm.es/api/bins/ajy3"
+
+
+async function getData() {
+  loading();
+  try {
+    const response = await fetch(url);
+    if (response.ok) {
+      const data = await response.json();
+      return data;
+    } else {
+      throw alert("apiデータが取得できません")
```

Comment:
Q
Why did you do the `throw alert`?

Reply:
> Why did you do the throw alert?
After handling the error, I thought it would be better to act on the user in a visible way.

Is it better to write the following?
`Error(response.statusText);`

commit : c526e1b